types.proto

```diff
@@ -32,13 +32,25 @@ message HashType {
     UNSPECIFIED = 0;
     SHA256 = 1;
     SHA512 = 2;
+    MD5 = 3;
   }
   HashMethod method = 1;
   bytes hash = 2;
 }
 
-// OpenConfig path representation of the target.
-// eg. /interfaces/interface[name=et-1/0/0]
+// Path encodes a data tree path as a series of repeated strings, with
+// each element of the path representing a data tree node name and the
+// associated attributes.
+// Reference: gNMI Specification Section 2.2.2.
 message Path {
```

Comment:
yes we should but i didn't want to block on that as it is a simple change once we figure out what that means